cli/tests/test_describe_command.rs

```diff
@@ -781,6 +781,63 @@ fn test_describe_avoids_unc() {
     assert_eq!(edited_path, dunce::simplified(&edited_path));
 }
 
+#[test]
+fn test_describe_opens_editor_if_edit_flag_overrides_default_behaviour() {
```

Comment:
question: What "default" behavior is this overriding?

Reply:
The default behaviour of not opening the editor if e.g. `--message` is passed.  But I agree that the name isn't great, what would you think about: `test_describe_with_edit_and_message_args_opens_editor()`?  That describes the contents of the test more accurately.

Reply:
Now using `test_describe_with_edit_and_message_args_opens_editor`